lua/urlview/pickers.lua

```diff
@@ -41,10 +40,17 @@ function M.telescope(items, opts)
       sorter = conf.generic_sorter(opts),
       attach_mappings = function(prompt_bufnr, _)
         actions.select_default:replace(function()
-          local selection = action_state.get_selected_entry()
+          local state = require "telescope.actions.state"
```

Comment:
Let's keep the action_state require from before, remove this line and rename the references below

Reply:
Done. Thank you for looking into it.

Reply:
No worries, thanks for making the change! Just ran the CI checks and there seems to be some minor formatting changes left :D

Reply:
I think it's good now

Reply:
Not sure if the changes made it 😅

Reply:
Sorry I just assumed there was a space missing but indentation was wrong too.